lib/sdt_alloc.bpf.c

```diff
@@ -444,7 +444,7 @@ int scx_alloc_free_idx(struct scx_allocator *alloc, __u64 idx)
 	if (likely(!data)) {
 
 		data[pos] = (struct sdt_data) {
-			.tid.gen = data->tid.gen + 1,
+			.tid.genn = data->tid.genn + 1,
```

Comment:
What's with `s/gen/genn/`? Did `gen` become a reserved token?

Reply:
Yep. Everything w/ codegen based ffi and garbage collection is gonna have fun w/ gen becoming a keyword.